Octokit/Models/Request/NewTeamDiscussion.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Used to create a team discussion.
+    /// </summary>
+    /// <remarks>
+    /// OAuth access tokens require the write:discussion scope.
+    /// See the <a href="https://developer.github.com/v3/teams/discussions/#create-a-discussion">Create a discussion</a> for more information.
+    /// </remarks>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class NewTeamDiscussion
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NewTeamDiscussion"/> class.
+        /// </summary>
+        /// <param name="title">The discussion post's title.</param>
+        /// <param name="body">The discussion post's body text.</param>
+        public NewTeamDiscussion(string title, string body)
+        {
+            Title = title;
+            Body = body;
+        }
+
+        /// <summary>
+        /// The discussion post's title (required).
+        /// </summary>
+        public string Title { get; set; }
```

Comment:
Mandatory field which gets set via the `ctor`, so this property should be `private set`

```suggestion
        public string Title { get; private set; }
```